src/media/js/site/components/header.js

```diff
@@ -39,24 +40,59 @@ class HeaderLogin extends React.Component {
     loginHandler: React.PropTypes.func.isRequired,
     logoutHandler: React.PropTypes.func.isRequired,
     isLoggedIn: React.PropTypes.bool.isRequired,
+    showingUserDropdown: React.PropTypes.bool.isRequired,
+    toggleUserDropdown: React.PropTypes.func.isRequired,
   };
+
+  renderAnonNav() {
+    return (
+      <nav className="header--nav">
+        <ul>
+          <li><LoginButton isSignup={true} {...this.props}/></li>
+          <li><LoginButton {...this.props}/></li>
+        </ul>
+      </nav>
+    );
+  }
+
+  handleLogoutClick = evt => {
+    this.props.toggleUserDropdown();
+    this.props.logoutHandler(evt);
+  }
+
+  renderUserNav() {
+    let dropdownClasses = classNames({
+      'header--nav': true,
+      'header--nav--show-user-dropdown': this.props.showingUserDropdown
+    });
+
+    return (
+      <nav className={dropdownClasses}>
```

Comment:
this entire dropdown seems redundant, what's the future plan for this because it just shows the same thing as the header while hiding stuff.